src/ViewInstance.js

```diff
@@ -657,6 +664,19 @@ class ViewInstance extends React.Component {
     const canCreateOrder = !checkIfUserInCentralTenant(stripes) && stripes.hasInterface('orders') && stripes.hasPerm('ui-inventory.instance.createOrder');
     const canReorder = stripes.hasPerm('ui-requests.reorderQueue');
     const canExportMarc = stripes.hasPerm('ui-data-export.app.enabled');
+
+    const hasSetForDeletionPermission = stripes.hasPerm(setForDeletionAndSuppressPerm);
+    const canCentralTenantSetForDeletion = hasSetForDeletionPermission && isShared;
```

Comment:
But all instances are shared on the central tenant, aren't they? Then why do we need to check for it? Maybe let's change for: `isCentralTenant && hasSetForDeletionPermission`?